extractor.js

```diff
@@ -170,7 +173,8 @@ function parseRegionsFile(buffer) {
 			}
 		};
 
-		region.id = stream.readUInt32BE();
+		region.full_id = stream.readUInt32BE();
```

Comment:
If possible I would like to avoid any breaking changes here. It would be preferable if the new data was given a field with a new name, like `individual_id` or `short_id` or something like that

Reply:
I'll use `individual_id`